android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,7 +117,38 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle = "";
+                    String track = "";
+                    String color = null;
```

Comment:
Hmm Sorry, my mistake, local variables don't have a default type, but I suppose Date start can be moved inside, is it used outside the loop too?